appunite/Loudius
# [LD-107] Extract mock-web-server-module.

# Pull-Request
## Description

### Why?

We extracted MockWebServerRule to external library https://github.com/appunite/MockWebServer-Extensions.

### What?

Removed code related to the MockWebServerRule and added [MockWebServer-Extensions](https://github.com/appunite/MockWebServer-Extensions).

## Links to related issues

- Fixes [LD-107](https://loudius.atlassian.net/browse/LD-107)


## Demo

Nothing to show :)

## How to test

You can see the [Run UI tests on Firebase Test Lab](https://github.com/appunite/Loudius/actions/runs/9005792363/job/24741906660?pr=216#logs) job passed.

## Documentation
- Link to library: https://github.com/appunite/MockWebServer-Extensions

## Checklist


- [x] Functionality is covered by unit tests
- [x] Functionality is covered by integration tests
